torch/_inductor/lowering.py

```diff
@@ -5767,6 +5769,93 @@ def resize_storage_bytes_(variable, new_size):
     return variable
 
 
+def resize_impl(self, size, *, memory_format=None):
+    if memory_format == torch.preserve_format:
+        raise RuntimeError(f"unsupported memory format: {memory_format}")
+
+    assert isinstance(self, TensorBox)
+    assert isinstance(size, (list, tuple))
+
+    new_numel = sympy_product(size)
+    old_numel = self.get_numel()
+    dtype = self.get_dtype()
+    device = self.get_device()
+    self_flat = view(self, (-1,))
+    new_strides = make_strides_for(size, memory_format)
+
+    if V.graph.sizevars.statically_known_leq(new_numel, old_numel):
+        sliced = slice_(self_flat, 0, 0, new_numel)
+        as_strided_(sliced, size, new_strides)
+        return sliced
+    else:
+        # We are growing the tensor, so we must preserve elements, and handle initialization of new ones
+        if (
+            torch.are_deterministic_algorithms_enabled()
+            and torch.utils.deterministic.fill_uninitialized_memory  # type: ignore[attr-defined]
+        ):
+            # deterministic fill value is documented, except for bool, but this seems to be consistant with eager
+            if is_float_dtype(dtype):
+                uninitalized_val = float("nan")
+            elif is_integer_dtype(dtype):
+                uninitalized_val = torch.iinfo(dtype).max
+            else:
+                uninitalized_val = True
+        else:
+            # using zero as that is what empty does
+            uninitalized_val = 0.0
+
+        self_loader = self_flat.make_loader()
+
+        def inner_fn(idx):
+            flat_index = ops.index_expr(idx[0], torch.int64)
+            limit = ops.index_expr(old_numel, torch.int64)
+            return ops.where(
+                ops.lt(flat_index, limit), self_loader(idx), uninitalized_val
+            )
+
+        pointwise = Pointwise.create(
+            device=device,
+            dtype=dtype,
+            inner_fn=inner_fn,
+            ranges=(new_numel,),
+        )
+        as_strided_(pointwise, size, new_strides)
+        return pointwise
+
+
+@register_lowering(aten.resize, type_promotion_kind=None)
+def resize(x, size, *, memory_format=None):
+    if memory_format is None:
+        memory_format = torch.contiguous_format
+
+    return resize_impl(x, size, memory_format=memory_format)
+
+
+@register_lowering(aten.resize_, type_promotion_kind=None)
+def resize_(x, size, *, memory_format=None):
```

Comment:
I'm not sure exactly what errors you're seeing but you probably need to add the resize ops to this list:
https://github.com/pytorch/pytorch/blob/7d05c4c093ad980d8c73d6af552fd603c321f6d1/torch/_inductor/graph.py#L959-L970